.changeset/ninety-wasps-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Update and add stories where Dropdown, Interactive Graph, Numeric Input, Expression, and Radio widgets render using answerless data
```

Comment:
For future PRs, it would be beneficial to have separate PRs. In this case it could have been 5 PRs.

Reply:
For sure. I thought each was small enough to be combined into one, but I can still separate them out if you think that would be useful?

Reply:
The main goal (IMO) of small PRs is to make review easier. Since you already have the reviews, I don't think it'd be worth the trouble.